cli/tests/fmt_check_tests_dir.out

```diff
@@ -1,2 +1,2 @@
 [WILDCARD]
-error: Found 1 not formatted file
+Error checking: [WILDCARD]
```

Comment:
I think this might be a regression? Seems like there is a failure to format a file instead of reporting that a file is not formatted. WDYT?

Reply:
Oh yeah, I meant to comment about this. There is a file with a syntax error in the folder and previously it wouldn't return a non-zero exit code, but now it does (to match `deno lint`).